apps/backoffice-v2/src/pages/Entity/pdfs/case-information/pages/IndividualSanctionsPage/components/IndividualSanctionsItem/ValueOrNone.tsx

```diff
@@ -0,0 +1,30 @@
+import { valueOrFallback } from '@/common/utils/value-or-fallback/value-or-fallback';
+import { valueOrNone } from '@/pages/Entity/pdfs/case-information/utils/value-or-none';
+import { Typography, tw } from '@ballerine/react-pdf-toolkit';
+import { AnyChildren } from '@ballerine/ui';
+import { FunctionComponent } from 'react';
+
+interface IValueOrNoneProps {
+  value?: unknown;
+}
+
+const NONE_TEXT_HEX_COLOR = '#999999';
+const valueOrNoneTextColor = valueOrFallback(NONE_TEXT_HEX_COLOR, { checkFalsy: true });
+
+export const ValueOrNone: FunctionComponent<IValueOrNoneProps> = ({ value }) => {
+  return (
+    <Typography
+      styles={[
+        tw(
```

Comment:
Does `tw` allow the same patterns as `clsx`? i.e. `{ 'h-10': condition }`?

Reply:
Nope, just string.